packages/core/src/result.ts

```diff
@@ -513,7 +513,7 @@ class Result<R extends RecordShape = RecordShape> implements Promise<QueryResult
           connectionHolder
             .releaseConnection()
             .then(() =>
-              connection?.protocol()?.version
+              connection?.getProtocolVersion()
```

Comment:
Was the `?` after `.protocol()` redundant? Because looking into the implementations of `Connection`, this was effectively turned from `connection?.protocol()?.version` to connection?.protocol().version`.

Reply:
Redundancy. Probably I just kept chain `?` by accident.